ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ServiceToOpenAPIMapper.java

```diff
@@ -230,6 +266,16 @@ public static OASResult generateOAS(OASGenerationMetaInfo oasGenerationMetaInfo)
         }
     }
 
+    private static Optional<ModuleID> getServiceModuleId(ServiceDeclarationNode serviceDeclarationNode,
+                                                         SemanticModel semanticModel) {
+        Optional<Symbol> symbol = semanticModel.symbol(serviceDeclarationNode);
+        if (symbol.isEmpty()) {
+            return Optional.empty();
+        }
+        Optional<ModuleSymbol> module = symbol.get().getModule();
+        return module.map(ModuleSymbol::id);
```

Comment:
Can't we do this ?
```suggestion
        return symbol.map(Symbol::getModule).map(ModuleSymbol::id);
```

Reply:
Fixed via [5e382c9](https://github.com/ballerina-platform/openapi-tools/pull/1745/commits/5e382c9d1336b5cfed22a3ebfb0618b14f328a15)